geoportailv3/static/js/authentication/authenticationdirective.js

```diff
@@ -64,76 +48,15 @@ app.AuthenticationController = function($http, loginUrl, logoutUrl,
     'login' : null,
     'password' : null
   };
-
-  /**
-   * @type {angular.$http}
-   * @private
-   */
-  this.http_ = $http;
-
-
-  /**
-   * @type {string|undefined}
-   */
-  this['login'] = undefined;
-
-
-  /**
-   * @type {string|undefined}
-   */
-  this['email'] = undefined;
-
-
-  /**
-   * @type {string|undefined}
-   */
-  this['role'] = undefined;
-
-  /**
-   * @type {number|undefined}
-   */
-  this['roleId'] = undefined;
-
-  /**
-   * @type {string|undefined}
-   */
-  this['name'] = undefined;
-
-  /**
-   * @type {boolean}
-   */
-  this['isError'] = false;
-
-  this.getUserInfo();
 };
 
 
 /**
- * @param {Object} credentials Credentials.
  * @export
  */
-app.AuthenticationController.prototype.authenticate = function(credentials) {
-
-  var that = this;
-  var req = $.param({
-    'login': credentials['login'],
-    'password': credentials['password']
-  });
-  var config = {
-    headers: {'Content-Type': 'application/x-www-form-urlencoded'}
-  };
-  this.http_.post(this.loginUrl_, req, config).success(
-      function(data, status, headers, config) {
-        if (status == 200) {
-          that.getUserInfo();
-          that['isError'] = false;
-        } else {
-          that['isError'] = true;
-        }
-      }).error(
-      function(data, status, headers, config) {
-        that['isError'] = true;
-      });
+app.AuthenticationController.prototype.authenticate = function() {
+
+  this.appUserManager_.authenticate(this['credentials']);
```

Comment:
If you use `login` and `password` properties as I suggested it you'll have this here:

```js
this.appUserManager_.authenticate(this.login, this.password);
```